nautobot_golden_config/templates/nautobot_golden_config/goldenconfigsetting.html

```diff
@@ -15,10 +15,13 @@
       <td>Description</td>
       <td>{{ object.description|placeholder }}</td>
     </tr>
+    <tr>
```

Comment:
I think it makes sense to have 3 rows:
* Dynamic Group (link to dynamic group)
* Filter Query Logic  (json)
* Scope of Devices (link to devices within dynamic group)

Reply:
implemented in next commit.